docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.8"
+
+volumes:
+  pub:
+
+services:
+  package:
+    image: dart:2.17
+    tty: true
+    stdin_open: true
+    working_dir: /home/package
+    environment:
+      - MEILISEARCH_HOST=http://meilisearch:7700
+      - PUB_CACHE=/vendor/pub-cache
+    depends_on:
+      - meilisearch
+    links:
+      - meilisearch
+    volumes:
+      - pub:/vendor/pub-cache
```

Comment:
Just a question, I'm curious to know why you need it in env variable and also on volume?